tests_to_write.md

```diff
@@ -0,0 +1,130 @@
+# Tests to write
+
+This is a set of notes on tests that need to get written to ensure this repo / package 
+actually can follow a test-driven development framework. 
```

Comment:
<3 definitely.

In `setup.cfg` test-related requirements can be added to separate them from the runtime requirements with `pip install .[test]` grabbing everything needed for local development. I threw some basic ones in there for the `pytest` framework (and iirc, code-coverage). would love to contribute github-actions workflows to automate testing as part of PRs (CI, eventually CD).